hs_core/templates/search/search.html

```diff
@@ -0,0 +1,141 @@
+<form method="get" action=".">
```

Comment:
This file is used for doing the search. It is included in the url.py configuration.